Code/Framework/AzToolsFramework/Tests/EditorViewportOnlyDrawHelpersForSelectedEntityTests.cpp

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/UnitTest/TestTypes.h>
+#include <AzFramework/UnitTest/TestDebugDisplayRequests.h>
+#include <AzFramework/Viewport/CameraState.h>
+#include <AzTest/AzTest.h>
+#include <AzToolsFramework/UnitTest/AzToolsFrameworkTestHelpers.h>
+#include <AzToolsFramework/UnitTest/Mocks/MockEditorViewportIconDisplayInterface.h>
+#include <AzToolsFramework/UnitTest/Mocks/MockEditorVisibleEntityDataCacheInterface.h>
+#include <AzToolsFramework/UnitTest/Mocks/MockFocusModeInterface.h>
+#include <AzToolsFramework/ViewportSelection/EditorHelpers.h>
+
+namespace UnitTest
+{
+    class EditorViewportOnlyDrawHelpersForSelectedEntityFixture
+        : public ToolsApplicationFixture
+        , public AzFramework::EntityDebugDisplayEventBus::Handler
+    {
+    public:
+        inline static constexpr AzFramework::ViewportId TestViewportId = 2468;
+
+        void SetUpEditorFixtureImpl() override
+        {
+            AllocatorsTestFixture::SetUp();
```

Comment:
update: I don' think you should need to call this, I believe it will already be called the base fixture so you can remove it